Rover/AP_Arming.cpp

```diff
@@ -175,7 +175,7 @@ bool AP_Arming_Rover::parameter_checks(bool report)
     // success if parameter checks are disabled
     if ((checks_to_perform != ARMING_CHECK_ALL) && !(checks_to_perform & ARMING_CHECK_PARAMETERS)) {
         return true;
-    }
+    }  
```

Comment:
Remove whitespace.

Reply:
"You need one commit for the copter change and another for the AP_Arming one." - you meaning is - one file - one commit ? I think it would be better create new branch from master and doing it there.

Reply:
> "You need one commit for the copter change and another for the AP_Arming one." - you meaning is - one file - one commit ? I think it would be better create new branch from master and doing it there.

One commit per directory, roughly speaking.  So one commit for the AP_Arming directory, on commit for the Copter directory, with the commit messages prefixed for the directory.

You don't need to create another branch - you might like to find a nice git tutorial somewhere and run through it.  It's a rather powerful tool.

OTOH - if you just want to get things done and can see a way forward - do that :-)

Reply:
Well, I'll try fix in this branch. About testing - I can only check in SITL if this code performed.

Reply:
Is it good idea use : "git revert HEAD~2..HEAD " for  tidying up my last two commits ?